### ISSUE TITLE
Old vs. New Question

### ISSUE THREAD

**Issue by axa-dental-doll** [rollerderby/scoreboard#56]
Hi! I am new to using the Scoreboard software in general.  Our league currently has an older version and I am considering updating to the new one.  Besides having the Pivot drop down option, what are some of the other options/perks available with the new version?  Is it 'worth' updating or just leaving what I currently have?
A few months ago, I heard a lot of "it runs slower and seems to freeze/crash more often than the old version" chatter, but I haven't really heard that lately.
So I'm just wondering, which one to use.  If I do update and don't prefer it, can I go back to the older version?
Thank you!

**Comment by rogerharkavy** [comment]
Edit: Didn't realize I was getting this from GitHub and not SourceForge.

Hi, Axa! The current stable version is 3.9.0 and I have found it to be very
reliable. There is a 3.9.3 "alpha" version that is currently in testing.
There have been reports of sluggish performance and lockups but the
developers are working on resolving these issues. I'm going to guess that
you don't need the newer features that 3.9.3 offers, so it's probably best
to stick with 3.9.0.

There's a rundown of each release and the features that have been added on
the releases page:
https://github.com/rollerderby/scoreboard/releases

I also recommend reading through the Installation and Quick Start topics on
the wiki: https://github.com/rollerderby/scoreboard/wiki

Also, join the Facebook group when you get the chance.
